convert_hf_to_gguf.py

```diff
@@ -3040,6 +3040,11 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
         return [(self.map_tensor_name(name), data_torch)]
 
 
+@Model.register("Olmo1124ForCausalLM")
+class Olmo1124Model(Model):
+    model_arch = gguf.MODEL_ARCH.OLMO_1124
```

Comment:
This class is a lot simpler than `OlmoModel` because that model is using the wrong rope type and the `OlmoModel` compensates for it.